packages/blade/src/components/Icons/index.tsx

```diff
@@ -16,7 +16,10 @@ type ActionIconColors = `action.icon.${DotNotationColorStringToken<
 
 export type IconSize = 'large' | 'medium' | 'small' | 'xlarge' | 'xsmall' | 'xxsmall';
 export type IconProps = {
-  color: ActionIconColors | FeedbackIconColors | FeedbackActionIconColors;
+  /**
+   * Color token (not to be confused with actual hsla value)
+   */
+  color: ActionIconColors | FeedbackIconColors | FeedbackActionIconColors | 'currentColor'; // currentColor is useful for letting the SVG inherit color property from its container
```

Comment:
should we rather add suffix as `Token`? if it wasted your whole day to debug 😅

Reply:
That might be a good idea but it would be a breaking change so let's get some more opinions perhaps /cc @anuraghazra @chaitanyadeorukhkar

Reply:
@kamleshchandnani you mean. ActionIconColorToken?

Reply:
The `color` prop -> `colorToken`